<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="windowModality">
   <enum>Qt::WindowModality::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>595</width>
    <height>112</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Materias Primas</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton{background-color: rgb(6, 191, 249);}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="lbl_fecha_cad_de">
       <property name="text">
        <string>Fecha caducidad</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lbl_cantidad_de">
       <property name="text">
        <string>Cantidad</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="le_cantidad_de">
       <property name="inputMethodHints">
        <set>Qt::InputMethodHint::ImhDigitsOnly</set>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QDateEdit" name="de_fecha_cad_de">
       <property name="inputMethodHints">
        <set>Qt::InputMethodHint::ImhDate</set>
       </property>
       <property name="currentSection">
        <enum>QDateTimeEdit::Section::DaySection</enum>
       </property>
       <property name="displayFormat">
        <string>d/M/yyyy</string>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
       <property name="date">
        <date>
         <year>2025</year>
         <month>1</month>
         <day>1</day>
        </date>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_salir_de">
       <property name="text">
        <string>Salir</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_guardar_de">
       <property name="text">
        <string>Guardar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
